app/modules/settings/actions/__tests__/settings-login-user-test.js

```diff
@@ -0,0 +1,34 @@
+import { expect } from "chai"
+import * as sinon from "sinon"
+
+import {settingsLoginUser} from "../settings-login-user"
+import {settingsUpdateUserState} from "../settings-update-user-state"
+
+describe("settingsLoginUser", () => {
+  let dispatch, ipcRenderer, sendStub, onStub
+
+  before(() => {
```

Comment:
`before` is a hook that only runs at the beginning of this suite, whereas `beforeEach` is run before each test - not sure if this is a conscious decision but this might introduce some side effects in the stubs

Reply:
I think my intention was to avoid running this multiple times when I wrote the test, but I realized this might make the test pass when they're really failing since the stub's state is persisted across the tests so I have changed them to beforeEach and afterEach!